test/test-lib/api-helpers.ts

```diff
@@ -149,14 +149,14 @@ export function expectJwt(tokenOrJwt: string | AnyObject) {
 			: tokenOrJwt
 	) as TokenUserPayload;
 	expect(decoded).to.have.property('id').that.is.a('number');
-	expect(decoded).to.have.property('username').that.is.a.string;
+	expect(decoded).to.not.have.property('username').that.is.a.string;
 	expect(decoded).to.have.property('jwt_secret').that.is.a.string;
 	expect(decoded).to.have.property('authTime').that.is.a('number');
 	expect(decoded).to.have.property('iat').that.is.a('number');
 	expect(decoded).to.have.property('exp').that.is.a('number');
 	expect(decoded).to.have.property('actor').that.is.a('number');
-	expect(decoded).to.have.property('email').that.is.a.string;
-	expect(decoded).to.have.property('created_at').that.is.a.string;
+	expect(decoded).to.not.have.property('email').that.is.a.string;
+	expect(decoded).to.not.have.property('created_at').that.is.a.string;
 	expect(decoded).to.have.property('permissions').that.is.an('array');
 
```

Comment:
validJwtProps.forEach((key) => {
		expect(decoded).to.have.property(key);
		unexpectedKeys.delete(key);
	});